osu.Game/Screens/Play/Player.cs

```diff
@@ -472,6 +464,14 @@ private Drawable createOverlayComponents(IWorkingBeatmap working)
                         Anchor = Anchor.Centre,
                         Origin = Anchor.Centre
                     },
+                    BreakOverlay = new BreakOverlay(working.Beatmap.BeatmapInfo.LetterboxInBreaks, ScoreProcessor)
+                    {
+                        Clock = DrawableRuleset.FrameStableClock,
+                        ProcessCustomClock = false,
+                        Breaks = working.Beatmap.Breaks
+                    },
+                    // display the cursor above some HUD elements.
+                    DrawableRuleset.Cursor?.CreateProxy() ?? new Container(),
```

Comment:
I guess I understand why this change was made but I will note that it leads to functional differences, namely that the cursor will now be above *all* HUD elements, not just *some* of them:

| master | this PR |
| :-: | :-: |
| ![osu_2024-08-27_09-52-06](https://github.com/user-attachments/assets/63ccdaf5-492a-495e-8100-641df3e2476f) | ![osu_2024-08-27_09-52-43](https://github.com/user-attachments/assets/8656da53-1b5d-48c0-bb01-519ed2cf9828) |

(see top left area on the screenshots above - on the left, the cursor is *underneath* the translucent wedge, and on the right it is not).

Was *this* particular behaviour change intentional or not?

Reply:
Aware, and touched on this in the commit description of https://github.com/ppy/osu/pull/29615/commits/abdbe510b884f86e5e9a5d9f746b8bddc51caac6. I don't have a better solution right now. The visual design intention was to keep the cursor underneath the HUD though.